tools/cmake/toolchain/openwrt.cmake

```diff
@@ -7,7 +7,11 @@ set(CMAKE_CROSS_COMPILING ON)
 
 # Platform specific
 set(TARGET_PLATFORM             "openwrt" CACHE STRING "Target Platform")
+if (DEFINED ENV{PRPLMESH_PLATFORM_TYPE})
 set(TARGET_PLATFORM_TYPE        "$ENV{PRPLMESH_PLATFORM_TYPE}" CACHE STRING "Target Platform Type")
+else()
+set(TARGET_PLATFORM_TYPE        "generic" CACHE STRING "Target Platform Type")
+endif()
```

Comment:
As you mention in the commit message, after this change only "ugw" is used in TARGET_PLATFORM_TYPE. So there's no need to define it as generic, you can just as well leave it empty. So I don't think this change is needed.

Reply:
I was thinking that it might become useful for printing a message, but sure can remove.